test/csuite/random_abort/main.c

```diff
@@ -363,14 +378,20 @@ recover_and_verify(uint32_t nthreads)
         /*
          * Every alternative thread is operated on column-store table. Make sure that proper cursor
          * is used for verification of recovered records.
+         *
+         * FIXME: Temporarily turn off column store test.
          */
+        columnar_table = false;
+        cursor = row_cursor;
```

Comment:
I would suggest making these new lines an #else so that it is clear what gets removed:
```
#if 0
...
#else
    /* FIXME... */
    columnar_table = false;
    cursor = row_cursor;
#endif 
```